Makefile

```
@@ -19,7 +19,9 @@ ifeq (${IMAGE_TAG},)
 IMAGE_TAG := ${SHORT_COMMIT}
 endif
 
-IMAGE_TAG_NO_NETGO := $(IMAGE_TAG)+without-netgo
+IMAGE_TAG_NO_ADX := $(IMAGE_TAG)+without-adx
+IMAGE_TAG_NO_NETGO_NO_ADX := $(IMAGE_TAG)+without-netgo-without-adx
+IMAGE_TAG_ARM := $(IMAGE_TAG)+arm
```

Unfortunately, we had to revert the change that added `+` here because it's not compliant with docker build tags. https://github.com/onflow/flow-go/pull/5267

Let's use `-` for now. We can circle back later to find a way to handle this

```suggestion
IMAGE_TAG_NO_ADX := $(IMAGE_TAG)-without-adx
IMAGE_TAG_NO_NETGO_NO_ADX := $(IMAGE_TAG)-without-netgo-without-adx
IMAGE_TAG_ARM := $(IMAGE_TAG)-arm
```